src/main/java/org/terasology/thirst/ThirstUtils.java

```diff
@@ -21,12 +21,24 @@
 import org.terasology.thirst.component.ThirstComponent;
 
 /**
+ * This class contains utility methods for the Thirst module. Specifically, it includes a method that returns the thirst
+ * of a given entity.
+ *
  * @author Marcin Sciesinski <marcins78@gmail.com>
  */
 public final class ThirstUtils {
+    /**
+     * Default initializer for the class.
```

Comment:
Just a side note: this constructor is private so that no instances of the utility class can be created. This pattern is called the [Singleton](https://en.wikipedia.org/wiki/Singleton_pattern).